### DISCUSSION TITLE
Glass/Acrylic material renders black

### DISCUSSION THREAD

**Discussion by WadeMorrison** [google/model-viewer#2858] [Q&A] [answered]
Hello! I've been trying to mess some of the settings based off of the GLTF extensions (modeling in Blender then editing/adding extensions in Visual Code), and I wanted to try and make a glass material using their IOR, Transmission, and Volume extensions and I have got it to work in other previewers (even your Model Editor) but when I import it into model-viewer they show up black unless there is something else behind it. I've tried other HDR's as well and still has the same results. Any idea what may be causing this or what I could do to fix this? I greatly appreciate your help!

Edit: I've also tried other sample models from KhronosGroup like the _Iridescent Dish with Olives_, same results. 

![image](https://user-images.githubusercontent.com/77404706/136852180-d05fc352-af39-4ee6-bd5f-7406bc8c3551.png)

**Comment by elalish** [comment, marked as answer]
This was recently fixed in three.js  and will be part of our upcoming release. The editor works because it uses master, rather than our last release.

**Reply by WadeMorrison** [reply]
@elalish Ah okay, sounds good then. Thank you very much for the help!
